verifyFiles.py

```diff
@@ -187,6 +197,27 @@ def run_command(command,file_name,condition1,condition2,subject_name,root_dir):
         log_error("Command "+command+" failed.")
     return output
 
+
+#Method is_checksum_equal reads the checksum
+#and returns true if the checksums are matching.
+def is_checksum_equal(condition_one_checksums_after_file,condition_two_checksums_after_file,subject,file_name):
```

Comment:
This method is reading in the checksums file every time it needs to access a checksum. This is a performance killer. Unless you expect the checksums file to be very big (>=1GB), I think the best would be to read the file once and for all, and store the checksum values in, e.g., a dictionary for later use.